exchange-config.skel/taler.conf

```diff
@@ -2,6 +2,8 @@
 CURRENCY = MYCURRENCY
 CURRENCY_ROUND_UNIT = MYCURRENCY:0.01
 
+AML_THRESHOLD = MYCURRENCY:999999999
```

Comment:
Without this line, the setup of the `taler-exchange-insecure` container would fail.